src/TriggerBinding/SqlTriggerListener.cs

```diff
@@ -406,7 +410,27 @@ PRIMARY KEY (UserFunctionID, UserTableID)
             using (var createGlobalStateTableCommand = new SqlCommand(createGlobalStateTableQuery, connection, transaction))
             {
                 var stopwatch = Stopwatch.StartNew();
-                await createGlobalStateTableCommand.ExecuteNonQueryAsync(cancellationToken);
+                try
+                {
+                    await createGlobalStateTableCommand.ExecuteNonQueryAsync(cancellationToken);
+                }
+                catch (Exception ex)
+                {
+                    TelemetryInstance.TrackException(TelemetryErrorName.CreateGlobalStateTable, ex, this._telemetryProps);
+                    var sqlEx = ex as SqlException;
+                    if (sqlEx?.Number == 2714)
```

Comment:
It seems to me that the number of magic numbers in the repo re: exception numbers is rapidly increasing. Can we create constants for these values?

Reply:
Sure, although in this case the comment above describes what the number is so I didn't think it necessary. I'll do a separate PR to create constants though.